Src/Support/Google.Apis.Auth/OAuth2/Flows/AuthorizationCodeFlow.cs

```diff
@@ -196,8 +197,13 @@ public AuthorizationCodeFlow(Initializer initializer)
             // Add exponential back-off initializer if necessary.
             if (DefaultExponentialBackOffPolicy != ExponentialBackOffPolicy.None)
             {
-                httpArgs.Initializers.Add(new ExponentialBackOffInitializer(
-                    DefaultExponentialBackOffPolicy, () => new BackOffHandler(new ExponentialBackOff())));
+                var effectiveRetryPolicy = (DefaultExponentialBackOffPolicy & ExponentialBackOffPolicy.RecommendedOrDefault) == ExponentialBackOffPolicy.RecommendedOrDefault ?
```

Comment:
I don't currently follow this. Will see if it makes more sense to me later..

Reply:
I now understand it a bit better, but it still feels a bit opaque. No concrete suggestions right now other than potentially extracting it out to a method somewhere (I think we've got the same logic elsewhere, so we could possibly remove that duplication at the same time) - but have a think.